### PR TITLE
Bugfix/profile picture is broken when user has allocation

### PR DESCRIPTION
**Pull request by isaacrata**
Before:
image

After:
image

### CODE REVIEW

**Comment by adbatista** [comment]
can you add a screenshot?

**Review by halan — CHANGES_REQUESTED** [review]
Please, add at least two screenshots focusing before/after on the description.

**Comment by izass** [comment]
> can you add a screenshot?


Admin user allocated:
image

Normal user allocated:
image

unallocated user:
image

File: app/views/refills/_navigation.html.erb [hunk 1]
```diff
@@ -30,16 +30,17 @@
...
                 <%= current_user.name %>
               </span>
-              <br/>
-              <span>
-                <%= current_user.current_allocation %>
-              </span>
-            </div>
+            <% end %>
+            <span>
+              <%= current_user.current_allocation %>
```
[the remark this answers is not in the document]
Comment (izass) [reply app/views/refills/_navigation.html.erb:40]: It's a anchor if the current user is a admin https://github.com/Codeminer42/Punchclock/pull/27/files#diff-38f5bfb9156477ed82cd017cf083587a11b4bb046f25864eff016f2b1c5d88de

**Pull request merged**